### PR TITLE
feat: homepage added

### PR DESCRIPTION
**Pull request by Anish-Parkhi**
Added UI for the home page

image

### CODE REVIEW

File: src/components/Card/Card.jsx [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+    <>
+        <div className={styles.main_div}>
+            <div className={styles.card_div}
+            style={{backgroundImage:`url(${props.image})`}}
+            >
+                <div className={styles.card_div_heading}>
+                    <p>
+                        {
+                            props.title
+                        }
```
Comment (Malhardixit) [inline src/components/Card/Card.jsx]: nit:space

File: src/components/Card/Card.jsx [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+                                    >
+                                        <img src={``} />
+                                        {
+                                            item
+                                        }
+                                    </div>
+                                </>
+                            )
+                        })
+                    }
```
Comment (Malhardixit) [inline src/components/Card/Card.jsx]: formatting

File: src/components/Navbar/Topnav.module.css [hunk 3]
```diff
@@ -0,0 +1,75 @@
...
+/* MUI images styling */
+
+.coins_image{
+    width: 20px;
+    height: 20px;
+}
+
+.images{
+    font-size: 40px;
+}
```
Comment (Malhardixit) [inline src/components/Navbar/Topnav.module.css]: rem

**Review by Malhardixit — APPROVED** [review]
by mistake

**Review by Malhardixit — CHANGES_REQUESTED** [review]
Please do the changes

File: src/components/Navbar/Topnav.module.css (hunk 3 above)
Comment (Anish-Parkhi) [reply src/components/Navbar/Topnav.module.css]: done

File: src/components/Card/Card.jsx [hunk 4]
```diff
@@ -10,11 +10,10 @@ function Card(props) {
...
             style={{backgroundImage:`url(${props.image})`}}
             >
+                <div className={styles.card_div_content}>
                 <div className={styles.card_div_heading}>
-                    <p>
-                        {
-                            props.title
-                        }
+                    <p className={styles.card_div_p}>
+                        {props.title}
```
Comment (Malhardixit) [inline src/components/Card/Card.jsx:16]: name can be better 

className={styles.cardTitle}>

File: src/components/Card/Card.jsx [hunk 5]
```diff
@@ -50,13 +48,13 @@ function Card(props) {
 
                 <div className={styles.play_button_div}>
                     <IconButton>
-                        <PlayCircleIcon
+                        <PlayArrowRoundedIcon
                         className={styles.play_icon}
-                        style={{color:"#2E4B90",fontSize:"80px"}}
+                        style={{color:"white",fontSize:"70px"}}
```
Comment (Malhardixit) [inline src/components/Card/Card.jsx:53]: 7 rem

**Review by Malhardixit — APPROVED** [review]
LGTM

Squash and merge it to master

**Pull request merged**